instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -150,16 +149,15 @@ def _instrumented_requests_call(
                     result = call_wrapped()  # *** PROCEED
                 except Exception as exc:  # pylint: disable=W0703
                     exception = exc
+                    setattr(
```

Comment:
It would but the linter doesn't like that. Between doing this and add lint ignore rule or using setattr. I changed this to use a constant for the "_otel_status_code" string so setattr is justified now.

Reply:
Resolving. LMK if you have more thoughts on this.